src/components/Home/Map/IncidentFocusCard.js

```diff
@@ -50,10 +50,10 @@ export default function IncidentFocusCard({ id, zoomOnCluster }) {
 
       {incident?.categories
         ? incident.categories.map(cat => (
-          <Tag key={nanoid()}>
-            {cat.charAt(0).toUpperCase() + cat.slice(1).replaceAll('-', ' ')}
-          </Tag>
-        ))
+            <Tag key={nanoid()}>
+              {cat.charAt(0).toUpperCase() + cat.slice(1).replaceAll('-', ' ')}
+            </Tag>
+          ))
```

Comment:
I'm a little confused by what happened here, is this just prettier doing its thing and formatting on save?

Reply:
Yes is prettier, im not sure what was happening but on my machine it kept asking